login.html

```diff
@@ -26,9 +27,12 @@
                </ul>
             </div>
 			 
-            <center>
-               <p>navigation bar/burger nav</p>
-            </center>
```

Comment:
I've done the same thing on this page also as the navigation bar should be consistent. I have added a 'logo' placeholder which will eventually be changed to the music database's name.